terraform/domains/environment_domains/config/production.tfvars.json

```diff
@@ -0,0 +1,12 @@
+{
+  "hosted_zone": {
+    "claim-additional-teaching-payment.service.gov.uk": {
+      "front_door_name": "#FRONT_DOOR_NAME#",
```

Comment:
The template was not rendered here. Should we remove all the non review enviroment files?

Reply:
good idea, will do!